lib/ResizableBox.js

```diff
@@ -44,10 +44,10 @@ export default class ResizableBox extends React.Component<ResizableProps, State>
   onResize = (e: SyntheticEvent<>, data: ResizeCallbackData) => {
     const {size} = data;
     const {width, height} = size;
-
-    if (this.props.onResize) {
+    const {onResize} = this.props;
+    if (onResize) {
       e.persist && e.persist();
-      this.setState(size, () => this.props.onResize && this.props.onResize(e, data));
+      this.setState(size, () => onResize && onResize(e, data));
```

Comment:
Probably not necessary to have the truthy check then:

```suggestion
      this.setState(size, () => onResize(e, data));
```